content/docs/2.0.0/iofogctl/platform-yaml-spec.md

```diff
@@ -146,60 +113,49 @@ Multiple resources can be incorporated into a single YAML file using `---` as a
 
 ```yaml
 ---
-apiVersion: iofog.org/v1
+apiVersion: iofog.org/v2
 kind: ControlPlane
 metadata:
   name: buffalo
   namespace: default
 spec:
   iofogUser:
-    name: Serge
-    surname: Radinovich
-    email: serge@edgeworx.io
+    name: neha
+    surname: naithani
```

Comment:
this could be a lot of commits by slapping over each other's username
can we stick to John Doe user@example.com?

Reply:
updated